packages/opentelemetry-sampler-aws-xray/README.md

```diff
@@ -0,0 +1,63 @@
+# AWS X-Ray Remote Sampler
+
+> This component is still in development and has not been released as an npm package.
+
+component owner: @carolabadeer
+
+This module provides the remote/centralized sampler for AWS X-Ray.
+
+## Usage
+
+```js
+
+const { AWSXRayRemoteSampler } = require('@opentelemetry/sampler-aws-xray');
+const opentelemetry = require("@opentelemetry/sdk-node");
+const { Resource } = require("@opentelemetry/resources");
+const { BatchSpanProcessor} = require('@opentelemetry/sdk-trace-base');
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-grpc');
+const { AWSXRayPropagator } = require("@opentelemetry/propagator-aws-xray");
+const { AWSXRayIdGenerator } = require("@opentelemetry/id-generator-aws-xray");
+
+
+// Initialize resource, trace exporter, span processor, and ID generator
+const _resource = Resource.default().merge(new Resource({
+        [SemanticResourceAttributes.SERVICE_NAME]: "remote-sampler-app",
+    }));
+const _traceExporter = new OTLPTraceExporter();
+const _spanProcessor = new BatchSpanProcessor(_traceExporter);
+const _tracerConfig = {
+    idGenerator: new AWSXRayIdGenerator(),
+}
+
+const sdk = new opentelemetry.NodeSDK({
+        textMapPropagator: new AWSXRayPropagator(),
+        instrumentations: [
+            new HttpInstrumentation(),
+            new AwsInstrumentation({
+                suppressInternalInstrumentation: true
+            }),
+        ],
+        resource: _resource,
+        spanProcessor: _spanProcessor,
+        traceExporter: _traceExporter,
+        // add remote sampler
+        sampler: new AWSXRayRemoteSampler(),
+    });
+
+    sdk.configureTracerProvider(_tracerConfig, _spanProcessor);
+
+```
+
+For more details on setting up the global tracer provider to send traces to AWS X-Ray, refer to [this documentation](https://aws-otel.github.io/docs/getting-started/js-sdk/trace-manual-instr#setting-up-the-global-tracer).
+
+Please note that AWS Lambda does not support X-Ray remote sampling. 
```

Comment:
I think this will make the linter pass. :slightly_smiling_face: 
```suggestion
Please note that AWS Lambda does not support X-Ray remote sampling.
```